packages/store/src/devicesSlice/mocks.ts

```diff
@@ -36,7 +38,7 @@ export const mockState = {
       check_in_time_exact: checkInTimeExact,
       check_in_time_rounded: checkInTimeRounded,
       identity_data: { mac: defaultMacAddress },
-      status: 'accepted',
+      status: DeviceDeviceauth.status.ACCEPTED,
```

Comment:
also more as a reference for the future: the device status is usually based on `DEVICE_STATES` throughout the UI, but we could move this to be tied to the deviceauth based type if you prefer (or have the UI based device states based on the types and go from there)

Reply:
This thing is a leftover from trying to comfort typescript in tests. I don't have opinion over first option or second, so I'll remove it for now